wordpress-mobile/WordPress-iOS
# Update to CocoaPods 1.2.1

Closes #6496

Updates to the latest CocoaPods but removes Helpshift and Lookback as pods. 

To test:
1. Clean all the thing (derived data, etc).
2. Update to CocoaPods 1.2.1. I suggest using RVM if gems frighten you. Well, then, RVM frightens me.
3. Build and run the project in debug.
4. Make sure Helpshift works. Lookback is only enabled on internal & alpha builds so maybe install the alpha build from Buddybuild.
5. Run the unit tests.

Needs review: @nheagy, @SergioEstevao
